frontend/components/board.js

```diff
@@ -9,24 +12,8 @@ function Square(props) {
 
 module.exports = React.createClass({
   getInitialState: function() {
-    var squares = Array(100).fill(null);
-    var isBlack = true;
-    for(var i=0; i<squares.length; i++) {
-      if(i%10 == 0) {
-        isBlack = !isBlack
-      }
-
-      if(isBlack) {
-        squares[i] = "black";
-        isBlack = !isBlack;
-        continue;
-      }
-
-      squares[i] = "white";
-      isBlack = !isBlack;
-    }
     return {
-      squares: squares,
+      squares: Array(boardHeight*boardWidth).fill("white"),
```

Comment:
Maybe later we can find a way to have user input select how they want the square colors organized